contrib/IECoreUSD/src/IECoreUSD/USDScene.cpp

```diff
@@ -129,34 +129,63 @@ IECoreScene::PointsPrimitivePtr convertPrimitive( pxr::UsdGeomPoints points, pxr
 
 IECoreScene::PointsPrimitivePtr convertPrimitive( pxr::UsdGeomPointInstancer pointInstancer, pxr::UsdTimeCode time )
 {
-	pxr::SdfPathVector targets;
-	pointInstancer.GetPrototypesRel().GetTargets( &targets );
-
-	IECore::StringVectorDataPtr instancePaths = new IECore::StringVectorData();
-	auto &writable = instancePaths->writable();
-	writable.resize( targets.size() );
-	for (size_t i = 0; i < targets.size(); ++i)
-	{
-		writable[i] = targets[i].GetString();
-	}
-
 	pxr::VtVec3fArray pointsData;
 	pointInstancer.GetPositionsAttr().Get( &pointsData, time );
 	IECore::V3fVectorDataPtr positionData = DataAlgo::fromUSD( pointsData );
 	positionData->setInterpretation( GeometricData::Point );
-
 	IECoreScene::PointsPrimitivePtr newPoints = new IECoreScene::PointsPrimitive( positionData );
 
-	IECore::DataPtr orientationData = DataAlgo::fromUSD( pointInstancer.GetOrientationsAttr(), time );
-	newPoints->variables["orient"] = IECoreScene::PrimitiveVariable( IECoreScene::PrimitiveVariable::Vertex, orientationData );
+	// Per point attributes
 
-	IECore::DataPtr instanceIndicesData = DataAlgo::fromUSD( pointInstancer.GetProtoIndicesAttr(), time );
-	newPoints->variables["instanceIndex"] = IECoreScene::PrimitiveVariable( IECoreScene::PrimitiveVariable::Vertex, instanceIndicesData );
+	if( auto protoIndicesData = DataAlgo::fromUSD( pointInstancer.GetProtoIndicesAttr(), time ) )
+	{
+		newPoints->variables["prototypeIndex"] = IECoreScene::PrimitiveVariable( IECoreScene::PrimitiveVariable::Vertex, protoIndicesData );
+	}
 
-	IECore::DataPtr scalesData = DataAlgo::fromUSD( pointInstancer.GetScalesAttr(), time );
-	newPoints->variables["scale"] = IECoreScene::PrimitiveVariable( IECoreScene::PrimitiveVariable::Vertex, scalesData );
+	if( auto idsData = DataAlgo::fromUSD( pointInstancer.GetIdsAttr(), time ) )
+	{
+		newPoints->variables["instanceId"] = IECoreScene::PrimitiveVariable( IECoreScene::PrimitiveVariable::Vertex, idsData );
```

Comment:
Simply because in the previous review you requested that I matched the Gaffer default. As for why it's `instanceId` in Gaffer I'm not sure - perhaps to better distinguish it from the `prototypeIndex`? We can change this to `id` here if you would prefer and then add a user default in Gaffer to match.

Reply:
Perhaps because Arnold reserves "id" & translation gets ignored that this choice was made?